code/chapters/09-mapping.py

```diff
@@ -74,6 +74,77 @@
 # - Panel: allows you to create applications/dashboards
 #
 # ### GeoPandas explore
+import io
+import pandas as pd
+import requests
+
+pd.set_option('display.max_columns', None)
+
+CRS = 'EPSG:32630'
+
+def get_databuffer(uri, encoding='utf-8'):
+    """Download data from URI and returns as an StringIO buffer"""
```

Comment:
Great use of 'raw character strings', which is done roughly as

```
r"(test\string)", "test string")
```

in R

Reply:
OK forget that, it's a docstring right? https://pandas.pydata.org/docs/development/contributing_docstring.html

Reply:
Yup, these are docstrings which I thought trying to explain stuff it would make sense for these helper functions. 

I also had also used "pylint" to check the code before submission, which moans a lot in their absence